src/mono/mono/component/debugger-protocol.h

```diff
@@ -147,8 +147,8 @@ typedef enum {
 	MDBGPROT_CMD_ASSEMBLY_GET_METHOD_FROM_TOKEN = 12,
 	MDBGPROT_CMD_ASSEMBLY_HAS_DEBUG_INFO = 13,
 	MDBGPROT_CMD_ASSEMBLY_GET_CATTRS = 14,
-	MDBGPROT_CMD_ASSEMBLY_GET_CUSTOM_ATTRIBUTES = 15,
-	MDBGPROT_CMD_ASSEMBLY_GET_PEIMAGE_ADDRESS = 16,
+	MDBGPROT_CMD_ASSEMBLY_GET_PEIMAGE_ADDRESS = 15,
```

Comment:
What happened to GET_CUSTOM_ATTRIBUTES?

Are we allowed to reuse command indexes?

Reply:
Wrong merge, I was testing it on .net7. Fixed thanks.